app/[lang]/about/page.tsx

```diff
@@ -3,131 +3,116 @@ import Footer from 'components/Footer'
 import { Button } from 'shared'
 
 import { sectionsConfig } from 'config/sections'
+import { useTranslations } from 'hooks'
+
+const CONTRIBUTORS = [
+  { username: 'adamcjonas', url: 'https://twitter.com/adamcjonas' },
+  { username: 'afterburn', url: 'https://github.com/afterburn' },
+  { username: 'benalleng', url: 'https://github.com/benalleng' },
+  { username: 'ecurrencyhodler', url: 'https://twitter.com/ecurrencyhodler' },
+  { username: 'gbks', url: 'https://twitter.com/gbks' },
+  { username: 'satsie', url: 'https://twitter.com/satsie' },
+  { username: 'shaavan', url: 'https://github.com/shaavan' },
+  { username: 'vinayaksh42', url: 'https://github.com/vinayaksh42' },
+]
+
+function Divider() {
+  return <div className="my-12 h-[1px] w-full bg-white bg-opacity-25" />
+}
+
+export default function About({ params }) {
+  const t = useTranslations(params.lang)
 
-export default function About() {
   return (
-    <div className="w-full">
+    <div className="w-full bg-[#2A3B61]">
       <Topbar items={sectionsConfig.mainNav} />
 
       <div className="m-auto px-6 lg:w-9/12 2xl:w-1/2">
         <div className="flex flex-col justify-center text-white">
           <h1 className="px-2.5 pt-20 text-center text-6xl sm:text-7xl lg:text-8xl">
-            About
+            {t('about.title')}
           </h1>
-          <p className="mt-4 text-center text-3xl">
-            The year is 2139. The very last bitcoin is slated to be mined in two
-            weeks. For months, a final countdown has been taking place in
-            Satoshi Square for the last block when there will officially be no
-            more bitcoin issued.
-          </p>
 
-          <p className="mt-8 font-nunito text-xl">
-            And then suddenly, the network comes to a crawl. Instead of the
-            normal ten-minute block interval, blocks are coming in just a few
-            times a day, if not longer. Your mom’s bitcoin node stops syncing.
-            The media starts reporting on the situation. It’s everywhere on
-            social media.
-            <br />
-            <br />
-            The price of bitcoin is falling. There is a knock at your door, and
-            when you open it, a yellowed, age-weathered envelope has been left
-            for you. There’s no name or return address. What could be inside?
+          <p className="mt-4 text-center text-4xl">{t('about.subtitle')}</p>
+          <p className="mt-8 font-nunito text-xl">{t('about.intro')}</p>
+
+          <Divider />
+
+          <h3 className="text-5xl">{t('about.project.title')}</h3>
```

Comment:
The size of the heading is correct for the desktop view but for mobile view, it needs to be `36px` - `text-4xl` as per the [figma mockup ](https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=2323-26516&t=eHsCzdoNoh1RCi6l-0)